### PR TITLE
(do not merge) Upgrade to package:ffi 1.0.0

### PR DESCRIPTION
**Pull request by johnpryan**
Tested on 2.12.0-259.16.beta

### CODE REVIEW

File: ffi/system-command/linux.dart [hunk 1]
```diff
@@ -36,7 +36,5 @@ int system(String command) {
   // Invoke the command, and free the pointer.
   int result = systemP(cmdP);
 
-  free(cmdP);
```
Comment (dcharkes) [inline ffi/system-command/linux.dart:39]: `calloc.free(cmdP)`

Comment (johnpryan) [reply ffi/system-command/linux.dart:39]: thanks!

**Comment by mit-mit** [comment]
Hey @johnpryan I found a number of inconsistencies in the FFI samples (output from C and Dart not the same; structs should now be by value). Can you take a look at https://github.com/dart-lang/samples/pull/88 ?

**Comment by johnpryan** [comment]
SGTM

**Pull request closed without merging**